pylint/lint/pylinter.py

```diff
@@ -43,6 +43,15 @@ def _read_stdin():
     return sys.stdin.read()
 
 
+def _load_reporter_by_class(reporter_class):
```

Comment:
Could you add typing on new functions ? We're trying to add it to the codebase since we dropped python 3.5 :)